Package.swift

```diff
@@ -13,18 +13,37 @@ let package = Package(
         .library(name: "Fluent", targets: ["Fluent"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/fluent-kit.git", from: "1.45.0"),
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.91.1"),
+        .package(url: "https://github.com/vapor/fluent-kit.git", from: "1.48.0"),
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.94.1"),
     ],
     targets: [
-        .target(name: "Fluent", dependencies: [
-            .product(name: "FluentKit", package: "fluent-kit"),
-            .product(name: "Vapor", package: "vapor"),
-        ]),
-        .testTarget(name: "FluentTests", dependencies: [
-            .target(name: "Fluent"),
-            .product(name: "XCTFluent", package: "fluent-kit"),
-            .product(name: "XCTVapor", package: "vapor"),
-        ]),
+        .target(
+            name: "Fluent",
+            dependencies: [
+                .product(name: "FluentKit", package: "fluent-kit"),
+                .product(name: "Vapor", package: "vapor"),
+            ],
+            swiftSettings: swiftSettings
+        ),
+        .testTarget(
+            name: "FluentTests",
+            dependencies: [
+                .target(name: "Fluent"),
+                .product(name: "XCTFluent", package: "fluent-kit"),
+                .product(name: "XCTVapor", package: "vapor"),
+            ],
+            swiftSettings: swiftSettings
+        ),
     ]
 )
+
+var swiftSettings: [SwiftSetting] { [
+    .enableUpcomingFeature("ConciseMagicFile"),
+    .enableUpcomingFeature("ForwardTrailingClosures"),
+    .enableUpcomingFeature("ImportObjcForwardDeclarations"),
+    .enableUpcomingFeature("DisableOutwardActorInference"),
+    .enableUpcomingFeature("IsolatedDefaultValues"),
+    .enableUpcomingFeature("GlobalConcurrency"),
+    .enableUpcomingFeature("StrictConcurrency"),
+    .enableExperimentalFeature("StrictConcurrency=complete"),
```

Comment:
You did it the other way around. You should remove strict concurrency from here and add existential any, since this is for Swift 5.8.
Also you don't need to use the upcoming StrictConcurrency flag. They did upgrade the flag to an upcoming feature for Swift 5.10, but it's backward compatible.